### DISCUSSION TITLE
Can it be opened in the browser through localhost

### DISCUSSION THREAD

**Discussion by Vilhelm-Ian** [SeaQL/FireDBG.for.Rust#6] [Q&A]
Can it be opened in the browser through localhost. I don't want to use visual studio code. I use vim

**Comment by tyt2y3** [comment]
In theory yes, because pretty much everything is HTML/js/wasm. May be you can use vscode only as a viewer, not editor.
The vscode extension also has some latency advantages because the server and client is in the same process, whereas a standalone webapp needs to make HTTP requests.

**Reply by Vilhelm-Ian** [reply]
Thanks

**Reply by yugaego** [reply]
@tyt2y3 I had a similar question as a Emacs user. I'd love to give a try to this package in practice, but doubt it would convince me to install VSC for that.

Good luck with it anyway, it looks very promising and addresses many of the needs I sometimes have debugging code.

**Reply by tyt2y3** [reply]
Thank you. Umm... regarding installing software. Would that be preferable if we package the viewer as a standalone Tauri app?

**Reply by Vilhelm-Ian** [reply]
Definetily. I would install it instantly

**Reply by tyt2y3** [reply]
Okay, let us work on it then!

**Comment by arduano** [comment]
This would be fairly awesome even for VSCode users. I use 3 1080p monitors, and a single monitor isn't enough to fit all the debugging as well as the code comfortably. But opening things in a child window on another monitor would make this so much nicer :)
